care/facility/api/viewsets/patient.py

```diff
@@ -124,6 +124,7 @@ class PatientFilterSet(filters.FilterSet):
         method="filter_by_category",
         choices=CATEGORY_CHOICES,
     )
+    ration_card_category = filters.ChoiceFilter(choices=RationCardCategory.choices)
```

Comment:
default = RationCardCategory.NON_CARD_HOLDER

Reply:
That would mean, for all existing data, care user has voluntarily "filled" as NO CARD HOLDER. As per issue, it is a non-mandatory field. This would cause confusion whether the person has a card or not right?